tika-parsers/tika-parsers-standard/tika-parsers-standard-modules/tika-parser-cad-module/src/test/java/org/apache/tika/parser/dwg/DWGParserTest.java

```diff
@@ -29,166 +29,179 @@
-        } finally {
-            input.close();
-        }
-    }
-
-    @SuppressWarnings("deprecation")
-    private void testParserNoHeader(InputStream input) throws Exception {
-        try {
-            Metadata metadata = new Metadata();
-            ContentHandler handler = new BodyContentHandler();
-            new DWGParser().parse(input, handler, metadata);
-
-            assertEquals("image/vnd.dwg", metadata.get(Metadata.CONTENT_TYPE));
-
-            assertNull(metadata.get(TikaCoreProperties.TITLE));
-            assertNull(metadata.get(TikaCoreProperties.DESCRIPTION));
-            assertNull(metadata.get(TikaCoreProperties.CREATOR));
-            assertNull(metadata.get(TikaCoreProperties.SUBJECT));
-            assertNull(metadata.get(TikaCoreProperties.COMMENTS));
-            assertNull(metadata.get(TikaCoreProperties.RELATION));
-
-            String content = handler.toString();
-            assertEquals("", content);
-        } finally {
-            input.close();
-        }
-    }
-
-    @SuppressWarnings("deprecation")
-    private void testParserAlt(InputStream input) throws Exception {
-        try {
-            Metadata metadata = new Metadata();
-            ContentHandler handler = new BodyContentHandler();
-            new DWGParser().parse(input, handler, metadata);
-
-            assertEquals("image/vnd.dwg", metadata.get(Metadata.CONTENT_TYPE));
-
-            assertEquals("Test Title", metadata.get(TikaCoreProperties.TITLE));
-            assertEquals("Test Subject", metadata.get(TikaCoreProperties.DESCRIPTION));
-            assertEquals("My Author", metadata.get(TikaCoreProperties.CREATOR));
-            assertEquals("My keyword1, MyKeyword2", metadata.get(TikaCoreProperties.SUBJECT));
-            assertEquals("This is a comment", metadata.get(TikaCoreProperties.COMMENTS));
-            assertEquals("bejanpol", metadata.get(TikaCoreProperties.MODIFIER));
-            assertEquals("http://mycompany/drawings", metadata.get(TikaCoreProperties.RELATION));
-            assertEquals("MyCustomPropertyValue",
-                    metadata.get(DWGParser.DWG_CUSTOM_META_PREFIX + "MyCustomProperty"));
-
-            String content = handler.toString();
-            assertContains("This is a comment", content);
-            assertContains("mycompany", content);
-        } finally {
-            input.close();
-        }
-    }
-
-    @Test
-    public void testAC1027() throws Exception {
-        Metadata metadata = getXML("testDWG-AC1027.dwg").metadata;
-        assertEquals("hlu", metadata.get(TikaCoreProperties.MODIFIER));
-    }
-
-    @Test
-    public void testAC1032() throws Exception {
-        Metadata metadata = getXML("testDWG-AC1032.dwg").metadata;
-        assertEquals("jlakshvi", metadata.get(TikaCoreProperties.MODIFIER));
-        assertEquals("CUSTOMER'S ADDRESS", metadata.get("dwg-custom:CUSTOMER'S ADDRESS"));
+    public void testDWGRead() throws Exception {
```

Comment:
If the executable is required, we should "assume" that it exists at the start of the test. Along the lines: https://github.com/apache/tika/blob/main/tika-parsers/tika-parsers-standard/tika-parsers-standard-modules/tika-parser-ocr-module/src/test/java/org/apache/tika/parser/ocr/TesseractOCRParserTest.java#L66